src/plugins/data/common/constants.ts

```diff
@@ -43,4 +43,6 @@ export const UI_SETTINGS = {
   FILTERS_EDITOR_SUGGEST_VALUES: 'filterEditor:suggestValues',
   AUTOCOMPLETE_USE_TIMERANGE: 'autocomplete:useTimeRange',
   AUTOCOMPLETE_VALUE_SUGGESTION_METHOD: 'autocomplete:valueSuggestionMethod',
+  MAX_BUCKETS_SETTING: 'metrics:max_buckets',
```

Comment:
Why are we defining these contants in the data plugin ? They are TSVB specific settings 🤔

Reply:
It seems to me I confused @DziyanaDzeraviankina  with my comment https://github.com/elastic/kibana/pull/110571#discussion_r700192617 =( . I wanted to create UI_SETTINGS object to follow common practice. Of course it should not affect data plugin